evap/development/fixtures/test_data.json

```diff
@@ -130235,6 +130235,69 @@
     "cc_users": []
   }
 },
+{
+  "model": "evaluation.userprofile",
+  "fields": {
+    "password": "",
+    "last_login": null,
+    "is_superuser": false,
+    "email": "oma.abner@institution.duplicate.com",
+    "title": null,
+    "first_name": "Oma",
+    "last_name": "Abner",
+    "language": null,
+    "is_proxy_user": false,
+    "login_key": null,
+    "login_key_valid_until": null,
+    "is_active": true,
+    "groups": [],
+    "user_permissions": [],
+    "delegates": [],
+    "cc_users": []
+  }
+},
+{
+  "model": "evaluation.userprofile",
+  "fields": {
+    "password": "",
+    "last_login": null,
+    "is_superuser": false,
+    "email": "lulu.ackerman@institution.duplicate.com",
+    "title": null,
+    "first_name": "Lulu",
+    "last_name": "Ackermann",
+    "language": null,
+    "is_proxy_user": false,
+    "login_key": null,
+    "login_key_valid_until": null,
+    "is_active": true,
+    "groups": [],
+    "user_permissions": [],
+    "delegates": [],
+    "cc_users": []
+  }
+},
+{
+  "model": "evaluation.userprofile",
+  "fields": {
+    "password": "",
+    "last_login": null,
+    "is_superuser": false,
+    "email": "echo.andre@institution.duplicate.com",
```

Comment:
The usage of `example.com` is deliberate, as this domain is reserved for such purposes. We don't have the same guarantee for `duplicate.com`, so I'd try to stick to `example.com`. Maybe we can use `duplicate.example.com`?